src/components/Identity/processIdSyncs.js

```diff
@@ -10,14 +10,17 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 
-import { assign, cookie, fireDestinations } from "../../utils";
-import namespace from "../../constants/namespace";
+import { assign, fireDestinations } from "../../utils";
 
-const millisecondsPerHour = 60 * 60 * 1000;
+const ID_SYNC_CONTROL = "idSyncControl";
+const ID_SYNC_TIMESTAMP = "idSyncTimestamp";
```

Comment:
It looks like you are using `ID_SYNC_TIMESTAMP` and `MILLISECONDS_PER_HOUR` here in the `index.js` modules. I would move them to the `constants` module and reuse them.

Reply:
It can be a `constants.js` module within the `Identity` component if no one else uses those consts.